### PR TITLE
Modify model zoo

### PR DESCRIPTION
**Pull request by bryanyzhu**
This PR adds the reason why there is a small gap between our trained SlowFast models to the official implementation. 

We also tested their provided pre-trained [models](https://github.com/facebookresearch/SlowFast/blob/master/MODEL_ZOO.md) on the un-normalized videos, their performance is usually 0.1%~0.4% lower than our trained models.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1149-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1149/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1149-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1149/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by takatosp1** [comment]
Thanks for mentioning it! If the video is not re-sampled, then it can be seen as performing sort of a temporal augmentation, which will lead to a difference in performance.
I would not seen this as `the performance heavily depends on the frame rate`, but more like doing precise things as what the paper reported.

**Comment by bryanyzhu** [comment]
@takatosp1 Of course, PySlowFast is a great repository. Thank you very much for the feedback, and thank you for maintaining PySlowFast. 

The way you look at not re-sampling as "half" temporal augmentation is interesting, and it is helpful. We also discussed whether to re-sample the video or not. For now, we train all our models on the original videos without re-sampling for fair comparison to previous approaches. But I will look into the effect of "half" temporal augmentation and "whole" temporal augmentation, and maybe provide a complete comparison later.

Thanks again for bringing up the discussion. Looking forward to more awesome work from your team.
